oryx-tui/src/app.rs

```diff
@@ -52,17 +52,15 @@ pub struct App {
     pub packets: Arc<RwLock<Vec<AppPacket>>>,
     pub notifications: Vec<Notification>,
     pub section: Section,
-    pub data_channel_sender: kanal::Sender<([u8; RawPacket::LEN], TrafficDirection)>,
+    pub data_channel_sender: kanal::Sender<([u8; RawFrame::LEN], TrafficDirection)>,
     pub is_editing: bool,
     pub active_popup: Option<ActivePopup>,
     pub start_from_cli: bool,
 }
 
 impl App {
     pub fn new(cli_args: &ArgMatches) -> Self {
-        let packets = Arc::new(RwLock::new(Vec::with_capacity(
-            RawPacket::LEN * 1024 * 1024,
-        )));
+        let packets = Arc::new(RwLock::new(Vec::with_capacity(RawFrame::LEN * 1024 * 1024)));
```

Comment:
`packets` should be renamed `frames` 
also, in the `App` structs, maybe change the fields `packets` to `frames` for consistency, what do you think ?